style: apply /style-guide pass to models/reports (#2687)
wandb/docs: models/reports/collaborate-on-reports.mdx

---
description: Collaborate and share W&B Reports with peers, co-workers, and your team.
title: Collaborate on reports
keywords: ["share report", "report comments", "star report", "report collaboration", "edit conflict"]
---

This page describes ways to collaborate on reports with your team. Use these workflows to share results, gather feedback, and keep important reports accessible.

> The following sections describe how to share a report, edit it collaboratively, add comments, and star reports for quick access.

Reviewer: ```suggestion
You can share a report, edit it collaboratively, add comments, or star it for quick access.
```

- To generate a sharing link instead, click **Share**. Adjust the permissions for the link, then click **Copy report link**. Share the link with the member.

When viewing the report, click a panel to open it in full-screen mode. If you copy the URL from the browser and share it with another user, the panel opens directly in full-screen mode when they access the link.

## Edit a report

Multiple team members can edit the same report and publish changes when they're ready. When any team member clicks the **Edit** button to begin editing the report, W&B automatically saves a draft. Select **Save to report** to publish your changes.

If an edit conflict occurs, such as when two team members edit the report at once, a warning notification helps you resolve any conflicts.

<Frame>
    <img src="/images/reports/share-report.gif" alt="Report sharing modal for a report in a 'Public' project" max-width="90%" />
</Frame>

## Comment on reports

To leave a comment on a report, click **Comment**.

> To comment directly on a panel, hover over the panel, then click the comment button, which looks like a speech bubble.

Reviewer: If this is FontAwesome we could inline the icon.

<Frame>
    <img src="/images/reports/demo_comment_on_panels_in_reports.gif" alt="Adding a comment to a panel" />
</Frame>

## Star a report

> If your team has many reports, click **Star** at the top of a report to add it to your favorites. When viewing your team's list of reports, click the star in a report's row to add it to your favorites. Starred reports appear at the top of the list.

Reviewer: Nit: It is actually the FontAwesome open star icon, and click the closed star icon to remove the star.
